net/nimble/controller/include/controller/ble_ll.h

```diff
@@ -481,6 +481,13 @@ ble_ll_get_addr_type(uint8_t txrxflag)
     return BLE_HCI_ADV_OWN_ADDR_PUBLIC;
 }
 
+/* Convert usecs to ticks and round up to nearest tick */
+static inline uint32_t
+ble_ll_usecs_to_ticks(uint32_t usecs)
```

Comment:
I'd name it ble_ll_usec_to_ticks_round_up() or similar, just to make it clear that this function rounds up to nearest